src/main/java/org/activiti/cloud/qa/story/ProcessInstanceTasks.java

```diff
@@ -79,4 +85,13 @@ public void verifyProcessStatus() throws Exception {
         assertThat(resultingEvent.getEventType()).isEqualTo(EventType.TASK_COMPLETED);
         assertThat(resultingEvent.getTask().getId()).isEqualTo(currentTask.getId());
     }
+
+    @Then("the status of the process is changed to completed by querying")
```

Comment:
If we remove the last line from the user story file, this step definition implementation should be moved into "the status of the process is changed to completed"